### PR TITLE
Navigation - About screen

### PR DESCRIPTION
**Pull request by Ekaanth**
Screen Mockup:

# IOS

# Android

**Tests**

- [ ] Android
- [ ] iOS

### CODE REVIEW

**Comment by abhiShandy** [comment]
I don't plan to have all the icons on the header for v1 (#1)
Just focus on the vertical ellipsis

File: theme/colors.ts [hunk 1]
```diff
@@ -11,4 +11,5 @@ export const colors = {
   Error: "#E53935",
   Warning: "#fca311",
   Success: "#0ecf96",
+  iconColor: "#ffffff",
```
Comment (abhiShandy) [inline theme/colors.ts]: no need to add this for simple colors, just use "white"

File: theme/theme.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from "react-native";
```
Comment (abhiShandy) [inline theme/theme.ts]: there's no need for this right now. I'll move styles to a common theme file when they are actually used at several places.

File: components/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        <View style={styles.userLogo}></View>
+      </View>
+      <View style={styles.headerRight}>
+        <FontAwesomeIcon icon={faWifi} size={20} style={iconColor} />
+        <FontAwesomeIcon icon={faExpand} size={20} style={iconColor} />
+        <Menu>
+          <MenuTrigger>
+            <FontAwesomeIcon icon={faEllipsisV} size={20} style={iconColor} />
+          </MenuTrigger>
+          <MenuOptions customStyles={optionsStyles}>
```
Comment (abhiShandy) [inline components/Header.tsx]: I want to show just one option right now: About

Comment (abhiShandy) [reply components/Header.tsx]: Think about how will this navigate to the About screen

**Comment by Ekaanth** [comment]
UPDATE: 

Added About sample screen 

# Android :

 
# IOS
https://user-images.githubusercontent.com/22363352/202672397-5c04ce43-dafb-48d8-ad63-90b20543ac5a.mov

**Comment by abhiShandy** [comment]
It looks different on my iOS 14 Pro Max simulator

**Comment by Ekaanth** [comment]
> It looks different on my iOS 14 Pro Max simulator

can you try to clear the cache and node_module and try again,  I had the same issue.

**Comment by abhiShandy** [comment]
> > It looks different on my iOS 14 Pro Max simulator
> 
> can you try to clear the cache and node_module and try again, I had the same issue.

It still looks bad. I tried iPhone 13 with iOS 16.1

**Pull request merged**